beta/package.json

```diff
@@ -96,5 +97,10 @@
     "budget": null,
     "budgetPercentIncreaseRed": 10,
     "showDetails": true
+  },
+  "lint-staged": {
+    "*.{js,ts,jsx,tsx}": "yarn prettier",
+    "*.css": "prettier --write",
```

Comment:
Why two different commands for prettier for CSS and everything else?

Reply:
Changed